swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, age=0):
+        id = uuid.uuid4().int if id is None else id
+        self.id = id
+        self.condition = condition
+        self.age = age
+
+
+    def __repr__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+
+
+    def get_category(self):
```

Comment:
👍 Good idea keeping this robust, so that no matter if the class name changes this method will always work!